### PR TITLE
Include buildings in property list

### PR DESCRIPTION
**Pull request by jdtoombs**
Include buildings in property list in addition to them being grouped with the parcel - but do not show a folder for buildings

### CODE REVIEW

File: frontend/src/components/Table/Table.tsx [hunk 1]
```diff
@@ -96,6 +96,7 @@ export interface TableProps<T extends object = {}> extends TableOptions<T> {
   detailsPanel?: DetailsOptions<T>;
   footer?: boolean;
   hideToolbar?: boolean;
+  hideBuildingFolder?: boolean;
```
Comment (goemen) [inline frontend/src/components/Table/Table.tsx:99]: This table can be used for other list, not just for properties. Can you use a more generic name for this property

Comment (Fosol) [reply frontend/src/components/Table/Table.tsx:99]: Good suggestion.  Expanding rows should be a reusable generic implementation.

Comment (jdtoombs) [reply frontend/src/components/Table/Table.tsx:99]: Making a generic prop that lets you pass a function to determine specific case where to allow expansion

Comment (asanchezr) [reply frontend/src/components/Table/Table.tsx]: yeah, I agree this should be generic if it applies to all tables or create a new component just for this use case

File: frontend/src/components/Table/Table.tsx [hunk 2]
```diff
@@ -96,6 +96,8 @@ export interface TableProps<T extends object = {}> extends TableOptions<T> {
   detailsPanel?: DetailsOptions<T>;
   footer?: boolean;
   hideToolbar?: boolean;
+  // Restrict where you would like an expansion button to appear
+  restrictExpansion?: (val: any) => boolean;
```
Comment (asanchezr) [inline frontend/src/components/Table/Table.tsx:100]: I think I'd prefer if this was named in the positive. Something like `canRowExpand?: (row: any) => boolean`
What do you think? 
It's usually harder the other way: "is it not restrictExpansion that allows expansion?" _ugh_

Comment (asanchezr) [reply frontend/src/components/Table/Table.tsx:100]: Also I think your logic below might not reflect the "restrict" naming...

Comment (jdtoombs) [reply frontend/src/components/Table/Table.tsx:100]: I agree, was having trouble naming. Started with showFolderWhere -> allowExpansionWhere -> restrictExpansion lol. Changing now

**Pull request merged**